tests/ffi/modules/test.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+
+. ../common/prepare.sh
+
+export QM_HOST_REGISTRY_DIR="/var/qm/lib/containers/registry"
+export QM_REGISTRY_DIR="/var/lib/containers/registry"
+
+disk_cleanup
+prepare_test
+reload_config
+
+# Download ffi-tools container and push ffi-tools image into QM registry
+prepare_images
+
+# Run the ffi-tools container in qm vm
+run_container_in_qm ffi-qm
+
+# Get result message of './modprobe_module'
+msg=$(podman exec -it qm /bin/bash -c \
+               "podman exec ffi-qm ./modprobe_module 2>&1")
+
+# Check result message displays right.
+if grep -eq "modprobe: FATAL: Module ext4 not found in directory /lib/modules/*" "$msg"; then
+   if_error_exit "Module ext4 should not found in /lib/modules/ inside QM"
+   exit 1
```

Comment:
we have an "if_error_exit" above, should we still use exit 1 here?  (I never tested, just wondering)

Reply:
Thanks, Done!